### ISSUE TITLE
FightPandemic - DevOps - Remote Worldwide

### ISSUE THREAD

**Issue by letizia123** [TechnologyMasters/jobs#322]
### Salary Expectation

- Volunteering

### Benefits

Why Join Us?

- Expand your network. You get to work as a part of a innovative, global and diverse team.
- Be part of a movement. We strive for bold changes and envision to make a big impact. Bring your passion, energy and creativity to participate.
- Learn and grow. You get to continuously learn and develop your skills. You might join us because of your skills in business development, but are able to contribute to any team's work if you'd like.
- Be free. You have the freedom to set the hours you are able to work every week.
- Do good. It is a scientific fact that volunteering for a good cause improves your well being.

### Location

- Remote

## What You'll Do _(Job Description)_

As a DevOps Engineer volunteer at Fight Pandemics, you will:

- Manage the current code deployment pipelines.
- Ensure the safety and security of all our infrastructure and data.
- Own the platform budget and make sure that it is used in the most optimal way.
- Monitor and maintain all the production machines and services.
- Maintain all the cloud tools used on the production infrastructure.
- Recommend ways to improve the current setup.
- Support product squads with the relevant knowledge about the infrastructure evolution and needs.

## What You Need to Be Successful _(Skills)_

- You are passionate about what we are building and you want to contribute and not wait on the sideways.
- Relevant knowledge related to cloud virtualization platforms such as AWS or Google Cloud.
- Relevant knowledge with regards to containerization technologies such as Docker.
- Excellent scripting and automation skills.
- You enjoy writing robust infrastructure.
- Positive and solution-oriented mindset.
- You are a problem solver.
- You want to be part of something great that positively impacts society especially in times of crisis.

### Must Have

- Openness to learn new technologies.
- Experience working with Git and Github in teams.
- Experience working with a remote team.
- Open source experience.
- Developer platform/tool industry experience.

### Nice to Have

Experience working with our stack
- Frontend: React Hooks, Ant Design, Ant Design Mobile, styled-components.
- Backend: Fastify, MongoDB, Auth0, Google Maps API.
- DevOps: Docker

### Relevant Experiences

- Being passionate, curious and proactive

## About Our company

We are creating an altruistic platform that facilitates the connection and exchange of information between individuals and/or organizations that need help with those that are providing help and vice versa. The way that we are going to bootstrap our social media, is by listing thousands of projects and initiatives that are tackling COVID-19 pandemic.
We are looking for volunteer who want to join us in our fight against pandemics like COVID-19 and other crisis alike.

## How to apply

Please apply here -> https://apply.workable.com/fightpandemics/j/13836E3C74/

---

## Meta

- [ ] Full Time
- [ ] Part Time
- [ ] Contract
- [ ] Internship
- [x] Remote Worldwide
- [ ] Remote Regional
- [x] Remote OK

**Comment by theetrain** [comment]
Closing due to missing information.
